addons/account/models/account_move.py

```diff
@@ -3470,6 +3471,22 @@ def _creation_message(self):
             'in_receipt': _('Purchase Receipt Created'),
         }[self.move_type]
 
+    def _notify_by_email_prepare_rendering_context(self, message, msg_vals, model_description=False,
+                                                   force_email_company=False, force_email_lang=False):
+        render_context = super()._notify_by_email_prepare_rendering_context(
+            message, msg_vals, model_description=model_description,
+            force_email_company=force_email_company, force_email_lang=force_email_lang
+        )
+        if self.invoice_date_due:
+            amount_txt = _('%(amount)s due %(date)s',
+                           amount=format_amount(self.env, self.amount_total, self.currency_id, lang_code=render_context.get('lang')),
+                           date=format_date(self.env, self.invoice_date_due, date_format='MM / YY', lang_code=render_context.get('lang'))
```

Comment:
Screenshot from specs says:

![image](https://user-images.githubusercontent.com/41282536/151763285-03b7a7f9-5014-4e00-ac41-41b7cd0ee43b.png)

So not sure "MM / YY" is correct?
Unless I'm missing something, or you discussed it with the PO.

Reply:
Updated with "short" format